pandas/tests/plotting/test_datetimelike.py

```diff
@@ -778,6 +778,42 @@ def test_mixed_freq_irreg_period(self):
         irreg.plot()
         ps.plot()
 
+    def test_mixed_freq_shared_ax(self):
+        import matplotlib.pyplot as plt
```

Comment:
This matches the other tests, but `plt` should be available on `self` here, e.g. `self.plt.`